.tekton/roxctl-push.yaml

```
@@ -202,13 +231,13 @@ spec:
           value: task
         resolver: bundles
       when:
-      - input: $(params.hermetic)
-        operator: in
-        values:
-        - "true"
+      - input: $(params.prefetch-input)
+        operator: notin
+        values: [ "" ]
```

Let us tackle this end-to-end in the context of ROX-20482. 

```suggestion
      # TODO(ROX-20482): Enable dependency prefetching - this may need to be compared with prefetch-input instead
      - input: $(params.hermetic)
        operator: in
        values: [ "true" ]
```

---

I left TODO note around `prefetch-input` parameter in this PR. I believe that should be sufficient.

---

To be done in #8440 .